packages/api/src/external/shared/patient-demographics.ts

```diff
@@ -134,54 +141,64 @@ export function patientToLinkedDemoData(patient: Patient): LinkDemoData {
 }
 
 export function createAugmentedPatient(
-  existingPatient: Patient,
+  patient: Patient,
   linksDempgraphics: LinkDemoData[]
 ): Patient {
+  const patientDemographics = patientToLinkedDemoData(patient);
+  const hashedPatientAddresses = patientDemographics.addresses.map(createAddressString);
+  // TODO Is submitting lower case addresses to CQ / CW okay, or do we have to grab the original version?
```

Comment:
Important question: can we send normalized addresses (i.e. all lower case)? Or do we think matching algorithms expect case-specific matching?

Reply:
empirical q we can answer. Did we start on that document of empirical qs we can investigate by "fuzzing" epic endpoints?

Reply:
Not yet, but I will!

Reply:
CQ staging shows no issues but we will monitor on production in a controlled env.